cloudify_ansible/__init__.py

```diff
@@ -48,6 +48,17 @@ def wrapper(group_name=None,
     return wrapper
 
 
+def set_ansible_env_vars(ansible_env_vars=None):
+    ansible_env_vars = ansible_env_vars or {}
+    if constants.OPTION_HOST_CHECKING not in ansible_env_vars:
```

Comment:
Instead of may ifs you can use .get(constants.OPTION_HOST_CHECKING,False)

Reply:
Generally, in Python style, if we are not using a value, we don't use `get`.